views/partials/nav.ejs

```diff
@@ -0,0 +1,8 @@
+<nav>
+    <ul>
+        <li><a href="/fullEpisodes">Full Episodes</a></li>
+        <li><a href="/index">Home</a></li>
+        <li><a href="/contact">Contact Us</a></li>
+        <li><a href= "episode link...">Click HERE to watch the Full Episode</a></li>
+    </ul>
+</nav>
```

Comment:
I think this partial  including a nav menu is not used anywhere, so it would make sense to delete this file.  (Otherwise it's just more code for teammates to have to review, and would likely cause confusion in search results, etc)
Not a big deal though.  Looks ok, otherwise